### ISSUE TITLE
Add support for mode keystroke efficient hinting

### ISSUE THREAD

**Issue by wishfort36** [qutebrowser/qutebrowser#4136]

**Comment by The-Compiler** [comment]
So how does vimperator differentiate between you typing 1 and 11? Do you need to press enter to follow the hint after typing the number?

**Comment by wishfort36** [comment]
@The-Compiler Yes. Vimperator would just wait, as there are multiple choices left.

**Comment by The-Compiler** [comment]
...which means you're back to the same number of keystrokes, at least for the common case of two-digit hints :wink:

**Comment by The-Compiler** [comment]
Fair point. I'm still kind of sceptical that it's worth the complexity, but I might take a look whenever I look at hints again.
